### PR TITLE
Version 2.0

### PR DESCRIPTION
**Pull request by HardeepAsrani**
- Add WooCommerce & LearnDash Block Conditions: https://github.com/Codeinwp/otter-blocks/issues/561
- Add Block Conditions indicator to blocks: https://github.com/Codeinwp/otter-blocks/issues/599
- Add free conditions to Block Conditions: https://github.com/Codeinwp/otter-blocks/issues/604
- Add Flip Block: https://github.com/Codeinwp/otter-blocks/issues/535
- Add WooCommerce Builder: https://github.com/Codeinwp/otter-blocks/issues/617
- Simplify Section's Sizing Controls: https://github.com/Codeinwp/otter-blocks/issues/563
- Improve Web Vitals: https://github.com/Codeinwp/otter-blocks/issues/526
- Upgrade to Block Registration API V2: https://github.com/Codeinwp/otter-blocks/issues/557
- Count Animation: https://github.com/Codeinwp/otter-blocks/issues/558
- Upgrade to using block.json files for blocks: https://github.com/Codeinwp/otter-blocks/issues/532
- Allow SVG Icons in Icon List: https://github.com/Codeinwp/otter-blocks/issues/539
- Add Counting and Typing Animations: https://github.com/Codeinwp/otter-blocks/issues/558
- Sticky Block Extension: https://github.com/Codeinwp/otter-blocks/issues/560
- Fix fatal error when FS mode isn't set to direct: https://github.com/Codeinwp/otter-blocks/issues/546
- Allow reset in Section Max Width: https://github.com/Codeinwp/otter-blocks/issues/693
- Use CSS Variables in generated CSS: https://github.com/Codeinwp/otter-blocks/issues/649
- Reduce SCSS nesting: https://github.com/Codeinwp/otter-blocks/issues/541
- CSS Review: https://github.com/Codeinwp/otter-blocks/issues/525
- Update animate.css to latest version: https://github.com/Codeinwp/otter-blocks/issues/556
- Add Pricing Section Variation: https://github.com/Codeinwp/otter-blocks/issues/699
- Remove unnecessary post types from Posts block: https://github.com/Codeinwp/otter-blocks/issues/745
- Add Global Defaults Sync: https://github.com/Codeinwp/otter-blocks/issues/710
- Posts Block Revamp: https://github.com/Codeinwp/otter-blocks/issues/655
- Add ACF Field to Posts Block: https://github.com/Codeinwp/otter-blocks/issues/655
- Fix empty slider freezing the website
- Fix Widgets screen crashing if used alongside WooCommerce
-  Visibility Condition based on WooCommerce User Spent: https://github.com/Codeinwp/otter-blocks/issues/754 
-  Visibility Conditions based on Category: https://github.com/Codeinwp/otter-blocks/issues/752
-  Visibility Conditions based on Post Type: https://github.com/Codeinwp/otter-blocks/issues/750
-  Visibility Conditions based on Query String: https://github.com/Codeinwp/otter-blocks/issues/743

### CODE REVIEW

**Comment by HardeepAsrani** [comment]
@irinelenache can you take a final look to make sure things are all good?

**Comment by irinelenache** [comment]
@HardeepAsrani Tested the build and here are my findings:

- [ ] On the flip block, inverting sides doesn't work anymore and makes the box shadow disappear
- [ ] Also, the flip block doesn't have any settings for mobile, and doesn't look ok on mobile
desktop: https://vertis.d.pr/O6XxhI
mobile: https://vertis.d.pr/Qo3283
^ Can be checked here: https://uniqueblood.tastewp.com/2022/03/03/test-flip-block/

- [ ] Related to Woocommerce builder, the changes made in the builder are not preserved. If i disable the Woo builder and enable it again, the product is empty again and i have to start over.  https://vertis.d.pr/cagYai

- [ ] When i set the Count delay for the counting animation, the value isn't changed and i get an error in console https://vertis.d.pr/4DvHQX . Same for the speed, but i don't get any error in console

- [ ] The icon color doesn't work on svg icons https://vertis.d.pr/SNjXjh

- [ ] The image width in the Posts block doesn't work https://vertis.d.pr/MPR7Wx

You can check them on this instance: 
```
 Admin area URL: https://uniqueblood.tastewp.com/wp-admin 
 Username: irinel 
 Password: klT0_OKdg60 
```
I'll come back with another comment after i test the widgets behaviour 🚀

**Comment by HardeepAsrani** [comment]
>  Related to Woocommerce builder, the changes made in the builder are not preserved. If i disable the Woo builder and enable it again, the product is empty again and i have to start over. https://vertis.d.pr/cagYai

That's intentional.

>  The icon color doesn't work on svg icons https://vertis.d.pr/SNjXjh

That is normal. We can't make the color work on images.

@Soare-Robert-Daniel Can you check the rest?

File: inc/css/blocks/class-flip-css.php [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+								'value'   => 'boxShadowColor',
+								'default' => '#000',
+								'format'  => function( $value, $attrs ) {
+									$opacity = ( isset( $attrs['boxShadowColorOpacity'] ) ? $attrs['boxShadowColorOpacity'] : 50 );
+									return ( strpos( $value, '#' ) !== false && $opacity < 100 ) ? $this->hex2rgba( $value, $opacity / 100 ) : $value;
+								},
+							),
+						),
+						'condition'      => function( $attrs ) {
+							return isset( $attrs['boxShadow'] ) && ( ! isset( $attrs['isInverted'] ) || ( isset( $attrs['isInverted'] ) && false === $attrs['isInverted'] ) );
```
Comment (Soare-Robert-Daniel) [inline inc/css/blocks/class-flip-css.php:218]: @HardeepAsrani, why did you disable the shadow in frontend when the sides are inverted?

Comment (HardeepAsrani) [reply inc/css/blocks/class-flip-css.php]: @Soare-Robert-Daniel I didn't. You had: https://github.com/Codeinwp/otter-blocks/commit/05fc51293e51da6bad675181beca40e7df596605 - must be for some reason that you can't remember now. You can test it a bit to see if there are any issues this way.

**Comment by HardeepAsrani** [comment]
>  I don't know if this is a problem, but if i add the Sharing icons to the sidebar, the sharing link is the link of the oldest post on the blog https://vertis.d.pr/rzqqgO

I'll leave it as it is right now. Not a major block. Will think more about how to solve this.

**Comment by Soare-Robert-Daniel** [comment]
@irinelenache, ready to test again.

>  - [ ] The image width in the Posts block doesn't work https://vertis.d.pr/MPR7Wx

The image width will be limited by the container. You can make it smaller, but not bigger that the container. If you can not make it smaller, please put the info of the instance.

**Comment by irinelenache** [comment]
@HardeepAsrani @Soare-Robert-Daniel Tested again and the checked issues are fixed, thank you 🚀 

I'll come back with another comment after i test the compatibilities with older wp and otter versions 👍 

Do you have any other special behaviours that should be tested?

**Comment by HardeepAsrani** [comment]
@irinelenache I've fixed all the mentioned issues. Let me know if you find anything else or if the release is all set to go!

**Comment by irinelenache** [comment]
@HardeepAsrani Tested again and everything is fixed now, no more issues on my end 🚀

**Pull request merged**